src/components/search/SearchInterface.vue

```diff
@@ -299,7 +299,13 @@
         return this.orderedFacets.filter(facet => this.coreFacetNames.includes(facet.name));
       },
       moreFacetNames() {
-        return this.facetNames.filter(facetName => !this.coreFacetNames.includes(facetName));
+        return this.facetNames.filter(facetName => {
```

Comment:
This seems to be causing a regression where the tier-0 filter isn't removed after selecting a thematic-collection while the side-facets are switched off.

Reply:
Not from this change, but from a change to `queryUpdatesForFacetChanges` in the search store, but yes.